tools/jira_notificator.py

```diff
@@ -0,0 +1,627 @@
+    """
+
+    for user in jira.search_users(user=email):
+        if user.emailAddress == email:
+            return user
+    logger.warning(f"User was not found for email {email}.")
+    return None
+
+def get_qa_contact(issue: Issue) -> Optional[User]:
+    """
+    Retrieves the QA contact user from an issue.
+
+    Args:
+        issue (Issue): The JIRA issue to inspect.
+
+    Returns:
+        Optional[User]: The QA contact user if set, otherwise None.
+    """
+
+    qa_contact = issue.fields.customfield_12315948
+    if qa_contact:
+        return qa_contact
+    else:
+        logger.warning(f"Issue {issue.key} does not have a QA contact.")
+        return None
+
+def get_manager(jira: JIRA, user: User) -> Optional[User]:
+    """
+    Retrieves the manager of a given user via LDAP and matches them in JIRA.
+
+    Args:
+        jira (JIRA): JIRA API client instance.
+        user (User): The user whose manager is to be found.
+
+    Returns:
+        Optional[User]: The manager user if found, otherwise None.
+    """
+
+    ldap = LdapHelper()
+    manager_email = ldap.get_manager_email(user.emailAddress)
+    if manager_email:
+        manager = find_user_by_email(jira, manager_email)
+        if manager:
+            return manager
+        else:
+            logger.warning(f"Manager {manager_email} was not found in Jira.")
+    else:
+        logger.warning(f"Manager of {user.emailAddress} was not found in LDAP.")
+    return None
+
+def get_assignee(issue: Issue) -> Optional[User]:
+    """
+    Retrieves the assignee user from an issue.
+
+    Args:
+        issue (Issue): The JIRA issue to inspect.
+
+    Returns:
+        Optional[User]: The assignee user if set, otherwise None.
+    """
+
+    assignee = issue.fields.assignee
+    if assignee:
+        return assignee
+    else:
+        logger.warning(f"Issue {issue.key} does not have an Assignee.")
+        return None
+
+def create_assignee_notification_text(missing_contact: Contact, notified_assignees: list[User]) -> str:
+    """
+    Creates a notification text for assignees when a contact is missing.
+
+    Args:
+        missing_contact (Contact): The type of missing contact.
+        notified_assignees (list[User]): List of users to be notified.
+
+    Returns:
+        str: The complete notification message text.
+    """
+
+    message = ""
+    if missing_contact == Contact.QA_CONTACT:
+        message = f"The QA contact is missing."
+    elif missing_contact == Contact.TEAM_LEAD or missing_contact == Contact.MANAGER:
+        message = f"There has been no response from the QA contact and the {missing_contact.value} is not listed in Jira."
+    else:
+        raise ValueError(f"Unknown Contact value: {missing_contact}.")
+    
+    return (
+        f"{create_notification_title(NotificationType.ASSIGNEE)}"
+        f"{create_jira_comment_mentions(notified_assignees)}"
+        f"{message} Could you please help us identify someone who could review the issue?"
```

Comment:
```suggestion
        f"{message} Could you please help us identify someone who could verify the issue?"
```
the goal of this message is find appropriate contact to verify the bug